test/Microsoft.Sbom.Api.Tests/VersionSpecificPins/Version_4_0/InterfaceConcretionTests.cs

```diff
@@ -322,6 +329,7 @@ private class PinnedIConfiguration : IConfiguration
         public ConfigurationSetting<bool> EnablePackageMetadataParsing { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
         public ConfigurationSetting<string> SbomPath { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
         public ConfigurationSetting<string> SbomDir { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+        public ConfigurationSetting<string> ComplianceStandard { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
     }
 
     private class PinnedISettingSourceable : ISettingSourceable
```

Comment:
We should consider adding `PinnedIManifestGenerator` in a separate PR. I should have added it for the 3.0 tests but missed the fact that it's exposed in the Extensions project and document for customer access

Reply:
I've made a note of this and will address in a separate PR.